components/style/properties/properties.mako.rs

```diff
@@ -65,6 +65,54 @@ pub trait MaybeBoxed<Out> {
     fn maybe_boxed(self) -> Out;
 }
 
+
+/// Addresses bug 16518. This creates a separate struct for
+/// font_size_keyword and cached_system_font. The cached_system_font
+/// has been appropriately cfged to exist only in gecko mode.
+#[derive(Clone, Debug)]
+pub struct Stash {
+    /// font-size keyword values (and font-size-relative values applied
+    /// to keyword values) need to preserve their identity as originating
+    /// from keywords and relative font sizes. We store this information
+    /// out of band in the ComputedValues. When None, the font size on the
+    /// current struct was computed from a value that was not a keyword
+    /// or a chain of font-size-relative values applying to successive parents
+    /// terminated by a keyword. When Some, this means the font-size was derived
+    /// from a keyword value or a keyword value on some ancestor with only
+    /// font-size-relative keywords and regular inheritance in between. The
+    /// integer stores the final ratio of the chain of font size relative values.
+    /// and is 1 when there was just a keyword and no relative values.
+    ///
+    /// When this is Some, we compute font sizes by computing the keyword against
+    /// the generic font, and then multiplying it by the ratio.
+   pub font_size_keyword: Option<(longhands::font_size::KeywordSize, f32)>,
+   /// The cached system font. See longhand/font.mako.rs
+   #[cfg(feature = "gecko")]
+   pub cached_system_font: Option<longhands::system_font::ComputedSystemFont>
```

Comment:
I don't see any usage of `cached_system_font`, I think this can be completely removed.